<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="test"
    errors="0" failures="1" hostname="" id="" package="test" skipped="0"
    tests="1" time="28.37" timestamp="Tue, 11 Sep 2018 12:23:27 GMT">
  
    <testcase name="default e2e tests" classname="test" time="28.37" assertions="2">

      <failure message="Testing if element &lt;.hello&gt; is present.">    at Object.defaultE2eTests [as default e2e tests] (E:/Vue/gitVue/vue-app/test/e2e/specs/test.js:14:15)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)</failure>

    
    </testcase>
  

  

  
  </testsuite>
</testsuites>
